stackdriver/client.go

```diff
@@ -120,9 +127,15 @@ func (c *Client) getConnection(ctx context.Context) (*grpc.ClientConn, error) {
 	if len(c.url.Port()) > 0 {
 		address = fmt.Sprintf("%s:%s", address, c.url.Port())
 	}
-	conn, err := grpc.DialContext(ctx, address, dopts...)
+	//	if c.resolver != nil {
+	//		conn, err := grpc.DialContext(ctx, c.resolver.Scheme()+":"+address, dopts...)
+	//		c.conn = conn
+	//		return conn, err
+	//	} else {
+	conn, err := grpc.DialContext(ctx, "dns:"+address, dopts...)
```

Comment:
This will always use the DNS resolver. I think you want to use `c.resolver.Scheme()`, unless `c.resolver` is `nil`.